### DISCUSSION TITLE
Nora Bateson and Symmathessy

### DISCUSSION THREAD

**Discussion by matthewmccarthy11** [life-itself/community#685] [Research]
Is Nora Bateson's critique of 'systems', and her proposal of the word 'symmathessy', a reasonable string of ideas? What about her claim that 'art is the only way to truly describe the living complexity of the world'? 

https://norabateson.wordpress.com/2015/11/03/symmathesy-a-word-in-progress/

**Comment by asimong** [comment]
To be honest, one of the problems I find here is that because she has a famous father, people think it's worth talking about her ideas. Personally I don't.

**Comment by asimong** [comment]
Elucidation is in the mind of the viewer, I have no problem with anyone being elucidated. But I do have other problems with her style.

**Comment by matthewmccarthy11** [comment]
Yeah, she is trying to do something similar to Francis Heylighen I think, (focusing more on the relational and learning aspects of 'systems'), but doesn't use the right framework or words or context (and her style is not that great) Nonetheless I do think she has some meaningful critiques of 'systems' concepts, and about their tendency to reduce systems to 'parts' and 'wholes' alone

Here is the paper from Francis: 

https://cris.vub.be/ws/portalfiles/portal/79556854/Heylighen_RelationalAgency.pdf

**Comment by asimong** [comment]
> Is Nora Bateson's critique of 'systems', and her proposal of the word 'symmathessy', a reasonable string of ideas?

I would say, not unreasonable. However that is not what I would ask. I would ask, does it tell us collectively something we didn't know before? That is, is it actually original research? Most critiques, including hers, have some point of sense to them. including critique of her position and style! But what does the word symmathesy give us beyond the concept that learning together has more potential than learning alone?

> What about her claim that 'art is the only way to truly describe the living complexity of the world'?

I am always cautious about weasel words like "only"and "truly". For sure, it is possible to propose a "true" concept of living complexity that can only be described by what one defines as art (circular definition warning notwithstanding) but is there only one true way to describe the living complexity of the world? I would say, tongue in cheek, only if you are a dogmatic fundamentalist, bent on reducing the actual complexity of the world to something that fits into your own mindset.
